### PR TITLE
Add coh recommendations to Data Export CFN

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cfn-templates/data-exports-aggregation.yaml [hunk 1]
```diff
@@ -102,19 +110,24 @@ Conditions:
...
+  DeployCOHViaCFN:       !And [!Condition IsSourceAccount, !Condition ManageCOH,         !Condition RegionSupportsDataExportsViaCFN]
   DeployCUR2ViaLambda:   !And [!Condition IsSourceAccount, !Condition ManageCUR2,  !Not [!Condition RegionSupportsDataExportsViaCFN]]
   DeployFOCUSViaLambda:  !And [!Condition IsSourceAccount, !Condition ManageFOCUS, !Not [!Condition RegionSupportsDataExportsViaCFN]]
+  DeployCOHViaLambda:   !And [!Condition IsSourceAccount, !Condition ManageCOH,    !Not [!Condition RegionSupportsDataExportsViaCFN]]
   DeployCUR2Table:       !And [!Condition IsDestinationAccount, !Condition ManageCUR2]
   DeployFOCUSTable:      !And [!Condition IsDestinationAccount, !Condition ManageFOCUS]
-  DeployAnyExportViaLambda: !Or [!Condition DeployCUR2ViaLambda, !Condition DeployFOCUSViaLambda]
-  DeployAnyTable: !Or [!Condition DeployFOCUSTable, !Condition DeployCUR2Table]
+  DeployCOHTable:        !And [!Condition IsDestinationAccount, !Condition ManageCOH]
+  DeployAnyExportViaLambda: !Or [!Condition DeployCUR2ViaLambda, !Condition DeployFOCUSViaLambda, !Condition DeployCOHViaLambda]
```
Comment (petrokashlikov) [inline cfn-templates/data-exports-aggregation.yaml:129]: What is this new concept of any lambda?

Comment (iakov-aws) [reply cfn-templates/data-exports-aggregation.yaml:129]: If any Export needs to be done via lambda - then need to deploy role and lambda

**Review by petrokashlikov — CHANGES_REQUESTED** [review]
For whatever reasons, I can't comment inline for earlier commits, but need to change wording on lines 414,478,528 from "## Deploy {export_name} natively via CFN resource if we are regions that support native CFN" to "## Deploy {export_name} natively via CFN resource in the regions that support native CFN"
and on lines 503,557 from  "## Deploy {export_name} natively via CFN resource if we are regions that support native CFN" to "Deploy {export_name} via lambda due to missing cfn resource definition.  Substitute {export_name} with COH and FOCUS respectively.

File: cfn-templates/data-exports-aggregation.yaml [hunk 2]
```diff
@@ -502,6 +525,63 @@ Resources:
         RefreshCadence:
           Frequency: "SYNCHRONOUS"
 
+  ## Deploy COH natively via CFN resource if we are regions that support native CFN
```
Comment (iakov-aws) [inline cfn-templates/data-exports-aggregation.yaml:528]:
```suggestion
  ## Deploy COH natively via CFN resource in the regions that support native CFN  
```

File: cfn-templates/data-exports-aggregation.yaml [hunk 3]
```diff
@@ -502,6 +525,63 @@ Resources:
...
+            S3OutputConfigurations:
+              Overwrite: "OVERWRITE_REPORT"
+              Format: "PARQUET"
+              Compression: "PARQUET"
+              OutputType: "CUSTOM"
+        Name: !Sub '${ResourcePrefix}-coh'
+        RefreshCadence:
+          Frequency: "SYNCHRONOUS"
+
+  ## Deploy COH natively via CFN resource if we are regions that support native CFN
```
Comment (iakov-aws) [inline cfn-templates/data-exports-aggregation.yaml:557]:
```suggestion
  ## Deploy COH via lambda due to missing CFN resource definition
```

**Comment by iakov-aws** [comment]
> For whatever reasons, I can't comment inline for earlier commits, but need to change wording on lines 414,478,528 from "## Deploy {export_name} natively via CFN resource if we are regions that support native CFN" to "## Deploy {export_name} natively via CFN resource in the regions that support native CFN" and on lines 503,557 from "## Deploy {export_name} natively via CFN resource if we are regions that support native CFN" to "Deploy {export_name} via lambda due to missing cfn resource definition. Substitute {export_name} with COH and FOCUS respectively.

Great catch. Fixed comment. 
not sure why commenting did not worked but you can always create a PR for the branch.

File: cfn-templates/data-exports-aggregation.yaml [hunk 4]
```diff
@@ -451,8 +475,9 @@ Resources:
         RefreshCadence:
           Frequency: "SYNCHRONOUS"
 
+  # FOCUS
 
-  ## Deploy FOCUS natively via CFN resource if we are regions that support native CFN
+  ## Deploy Data Export natively via CFN resource if we are regions that support native CFN
```
Comment (petrokashlikov) [inline cfn-templates/data-exports-aggregation.yaml:480]: Still need to change wording to "## Deploy Data Export natively via CFN resource in regions that support native CFN"

File: cfn-templates/data-exports-aggregation.yaml [hunk 5]
```diff
@@ -502,6 +527,65 @@ Resources:
         RefreshCadence:
           Frequency: "SYNCHRONOUS"
 
+  # FOCUS
```
Comment (petrokashlikov) [inline cfn-templates/data-exports-aggregation.yaml:530]: This should be COH

File: cfn-templates/data-exports-aggregation.yaml [hunk 6]
```diff
@@ -502,6 +527,65 @@ Resources:
...
+
+  ## Deploy Data Export natively via CFN resource if we are regions that support native CFN
```
Comment (petrokashlikov) [inline cfn-templates/data-exports-aggregation.yaml:532]: Still need to change wording to "## Deploy Data Export natively via CFN resource in regions that support native CFN"

**Review by petrokashlikov — CHANGES_REQUESTED** [review]
Still need to change few wordings, as they remained the same

File: cfn-templates/data-exports-aggregation.yaml (hunk 5 above)
Comment (iakov-aws) [inline cfn-templates/data-exports-aggregation.yaml:530]:
```suggestion
  # COH
```

File: cfn-templates/data-exports-aggregation.yaml (hunk 4 above)
Comment (iakov-aws) [inline cfn-templates/data-exports-aggregation.yaml:480]:
```suggestion
  ## Deploy Data Export natively via CFN resource in regions that support native CFN
```

File: cfn-templates/data-exports-aggregation.yaml [hunk 7]
```diff
@@ -502,6 +527,65 @@ Resources:
         RefreshCadence:
           Frequency: "SYNCHRONOUS"
 
+  # COH
+
+  ## Deploy Data Export natively via CFN resource if we are regions that support native CFN
```
Comment (iakov-aws) [inline cfn-templates/data-exports-aggregation.yaml:532]:
```suggestion
  ## Deploy Data Export natively via CFN resource in regions that support native CFN
```

**Review by petrokashlikov — APPROVED** [review]
LGTM

**Pull request merged**